mirage/ramp_generator/obs_generator.py

```diff
@@ -2918,8 +2919,8 @@ def save_DMS(self, ramp, zeroframe, filename, mod='1b', err_ext=None,
         outModel.meta.observation.time = self.params['Output']['time_obs']
 
         # Create INT_TIMES table, to be saved in INT_TIMES extension
-        int_times = self.int_times_table(self.ramptime, self.params['Output']['date_obs'], self.params['Output']['time_obs'],
-                                         outModel.data.shape[0])
+        int_times = self.int_times_table(self.ramptime, '2022-08-27', '12:49:00.000', outModel.data.shape[0])
```

Comment:
Did you mean to leave this in?

Reply:
Same issue as above. I just need to make sure the `date_obs` is a string.